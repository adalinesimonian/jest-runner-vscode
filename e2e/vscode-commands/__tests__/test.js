const path = require('path')
const { commands, Uri, workspace, window } = require('vscode')

const docPath = path.resolve(__dirname, '../test.json')

describe('Describe', () => {
  it('should test', async () => {
    await expect(
      commands.executeCommand('vscode.openFolder', Uri.file(__dirname))
    ).resolves.toBeUndefined()

    const doc = await workspace.openTextDocument(docPath)
    await window.showTextDocument(doc)

    expect(doc.getText()).toMatchSnapshot()
  })
})
