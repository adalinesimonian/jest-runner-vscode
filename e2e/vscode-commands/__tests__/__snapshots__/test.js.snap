// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Describe should test 1`] = `
"{
  \\"x\\": 5
}
"
`;
